routers/web/web.go

```diff
@@ -872,7 +872,7 @@ func RegisterRoutes(m *web.Route) {
 						m.Delete("", repo.DeleteProjectBoard)
 						m.Post("/default", repo.SetDefaultProjectBoard)
 
-						m.Post("/{index}", repo.MoveIssueAcrossBoards)
+						m.Post("/{issueIndex}/{sorting}", repo.MoveIssueAcrossBoards)
```

Comment:
Sorry I was wrong ... it is really an issue ID, and used by `GetIssueByID` 😭

Reply:
no problem reverted it